AspNetCore.Demo.Library/Interfaces.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace AspNetCore.Demo.Library
+{
+    public interface IScriptGlobals
```

Comment:
I generally try to have file per type, so I would split those into separate files.

Reply:
Fixed!